packages/agent/conf/demos/knb/kvk/presentation_definitions/kvk.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "knb-kvk",
+  "purpose": "We hebben informatie nodig van u",
+  "input_descriptors": [
+    {
+      "id": "4cf7cff1-0234-4f35-9d21-251668a60950",
+      "name": "Verificatie notaris",
+      "purpose": "We hebben informatie verstrekt door uw notaris nodig",
```

Comment:
this does not read very nicely. something like:

We hebben inforrmatie nodig dat is verstrekt door uw notaris?

Reply:
We hebben de door de KvK verstrekte informatie nodig.